nx.json

```diff
@@ -26,6 +26,26 @@
     "e2e": {
       "dependsOn": ["^build"],
       "inputs": ["default", "^production"]
+    },
+    "version": {
+      "dependsOn": ["^build"],
+      "inputs": ["production", "^production"],
+      "executor": "@jscutlery/semver:version",
+      "options": {
+        "preset": {
+          "name": "conventionalcommits",
+          "commitUrlFormat": "{{host}}/{{owner}}/{{repository}}/commit/{{hash}}",
+          "compareUrlFormat": "{{host}}/{{owner}}/{{repository}}/compare/{{previousTag}}...{{currentTag}}",
+          "issueUrlFormat": "{{host}}/{{owner}}/{{repository}}/issues/{{id}}",
+          "types": [
+            { "type": "feat", "section": "Features" },
```

Comment:
what I am also using for dependency updates, nx changes or simillar is `build`, would it make sense to have it here too?

Reply:
there is `build`and `ci`

Reply:
BTW, Nx released now a new command `nx release` we should test it :)